src/js/letters/actions/letters.js

```diff
@@ -12,9 +12,25 @@ import {
   GET_LETTER_PDF_FAILURE,
   GET_LETTER_PDF_SUCCESS,
   LETTER_ELIGIBILITY_ERROR,
-  UPDATE_BENFIT_SUMMARY_REQUEST_OPTION
+  UPDATE_BENFIT_SUMMARY_REQUEST_OPTION,
+  UPDATE_ADDRESS_SUCCESS,
+  UPDATE_ADDRESS_FAILURE
 } from '../utils/constants';
 
+// Copied from the vets-api v0/address POST swagger; TODO: delete once
+// actual form data is passed to updateAddress
+const exampleAddress = {
+  type: 'DOMESTIC',
+  addressEffectiveDate: '1973-01-01T05:00:00.000+00:00',
```

Comment:
Same question here, we're not including an input for this field, so not sure what this value is going to be based on. Also, I've been referring to this ticket for which fields are required or not (https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4415) and `addressEffectiveDate` isn't mentioned, so I assume it's never required?

Reply:
👍  I'll set the addressEffectiveDate on the back end